packages/create-next-app/index.ts

```diff
@@ -357,15 +357,15 @@ async function run(): Promise<void> {
 
     if (!process.argv.includes('--app') && !process.argv.includes('--no-app')) {
       if (ciInfo.isCI) {
-        program.app = true
+        program.app = getPrefOrDefault('app')
```

Comment:
This looks like a bug since `app` does not define a default here:

https://github.com/joulev/next.js/blob/e10eb4b1c1aef6aa51ff170f893ab92933d53fe5/packages/create-next-app/index.ts#L245-L252